### PR TITLE
Kyrre/improve detection of stop place changes

### PR DESCRIPTION
**Pull request by tft-kyrre**
Improved detection of changes in imported data.

Ref: [https://github.com/TromsFylkestrafikk/rtm/issues/487](https://github.com/TromsFylkestrafikk/rtm/issues/487)

### CODE REVIEW

**Comment by tft-kyrre** [comment]
Not sure why those old issued appeared. Should they be removed? How?

**Comment by tfk-kaare** [comment]
This is probably due to a mismatch between local branch (with rebase) and remote branch (probably without). When pushing local branches (which are rebased) to remote, the `--force` option must be provided

**Comment by tft-kyrre** [comment]
The problem with old issues showing up was caused by me working on an outdated branch. Fixed now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+- Improved detection of changes in imported data.
```
Comment (tfk-kaare) [inline CHANGELOG.md:6]: In imported stops?

**Review by tfk-kaare — COMMENTED** [review]
Just a tiny bit.

This comparison/test looks awesome. So simple, so impactful.

File: CHANGELOG.md (hunk 1 above)
Comment (tft-kyrre) [reply CHANGELOG.md:6]: sure, but other models like `TariffZone` and `TopographicPlace` will also benefit from the improved comparison method.

**Review by tfk-kaare — APPROVED** [review]

**Pull request merged**